demo/build.gradle

```diff
@@ -45,6 +47,20 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
```

Comment:
Perhaps `minifyEnabled` should be set to `true` here?

Reply:
Done in https://github.com/googlemaps/android-maps-utils/pull/811/commits/60e9881cf269b168a82319dadfe20c02d951a0ff